state-chain/pallets/cf-elections/src/electoral_systems/mocks.rs

```diff
@@ -79,36 +83,48 @@ where
 	// Useful for testing check_consensus since we already have an election.
 	pub fn build_with_initial_election(self) -> TestContext<ES> {
 		let setup = self.clone();
-		let mut electoral_access = MockAccess::<ES>::new(
-			self.unsynchronised_state,
-			self.unsynchronised_settings,
-			self.electoral_settings,
-		);
+
```

Comment:
Is the clone of self (line above this) still necessary?
I believe it may have been there because the old `self` methods were mutating.

Reply:
removed the clone